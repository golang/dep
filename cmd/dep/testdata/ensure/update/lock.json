{
    "memo": "9a5243dd3fa20feeaa20398e7283d6c566532e2af1aae279a010df34793761c5",
    "projects": [
        {
            "name": "github.com/carolynvs/go-dep-test",
            "version": "0.1.0",
            "revision": "b9c5511fa463628e6251554db29a4be161d02aed",
            "packages": [
                "."
            ]
        },
        {
            "name": "github.com/pkg/errors",
            "branch": "v0.7.0",
            "revision": "01fa4104b9c248c8945d14d9f128454d5b28d595",
            "packages": [
                "."
            ]
        }
    ]
}
