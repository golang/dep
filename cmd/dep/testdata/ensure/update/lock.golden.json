{
    "memo": "9a5243dd3fa20feeaa20398e7283d6c566532e2af1aae279a010df34793761c5",
    "projects": [
        {
            "name": "github.com/carolynvs/go-dep-test",
            "version": "0.1.1",
            "revision": "40691983e4002d3a3f5879cc0f1fe99bedda148c",
            "packages": [
                "."
            ]
        },
        {
            "name": "github.com/pkg/errors",
            "branch": "v0.7.0",
            "revision": "01fa4104b9c248c8945d14d9f128454d5b28d595",
            "packages": [
                "."
            ]
        }
    ]
}
